scripts/generator/templates/template-release-notes-maintenance.md

```diff
@@ -27,7 +27,7 @@ Here are the highlighted new and improved features in this maintenance release.
 
 # Getting Started with Vaadin 14
 ## App starters
-The best way to get started with Vaadin 14 is to go to [https://vaadin.com/start](https://vaadin.com/start) and pick an app template for the technology stack you’re interested in.
+The best way to get started with Vaadin 14 is to go to [https://start.vaadin.com](https://start.vaadin.com) and pick an app template for the technology stack you’re interested in.
```

Comment:
> the technology stack you’re interested in.

We should rephrase this part as there isn't an option to choose a stack at https://start.vaadin.com (only Spring Boot).

Reply:
@web-padawan I already did while you wrote your review 😄

Reply:
Thanks!